Add release note for Gradle init changes (#27288)
gradle/gradle: platforms/documentation/docs/src/docs/release/notes.md

> Get more help at https://help.gradle.org.
```

### Gradle init

> When using `gradle init` to generate projects with Kotlin DSL scripts they are now using [simple property assignment](/8.4/release-notes.html#assign-stable) syntax with `=` operator.

Reviewer: ```suggestion
Projects generated using `gradle init` with Kotlin DSL scripts now use [simple property assignment](/8.4/release-notes.html#assign-stable) syntax with the `=` operator.
```
